ReactTask/src/components/CalendarVisitDoctor/CalendarVisitDoctor.js

```diff
@@ -67,7 +67,7 @@ const CalendarVisitDoctor = ({ changeData }) => {
           .fill(null)
           .map((_, index) => {
             const daySelect = index - (startDay - 2);
-            changeData(daySelect);
```

Comment:
line 22-23 this file. 
const today = new Date();
  const [date, setDate] = useState(today);
You could simply const [date, setDate] = useState(new Date());

Reply:
Fixed